docs/SKILLS.md

```diff
@@ -0,0 +1,35 @@
+# Skill system specification
+
+## Glossary
+
+Skill - a player's skill that he can improve throughout the game until its level becomes maximum.
+
+## Variables
+
+### Skill properties
+
+- `maxLevel` - maximum level for this skill
+- `baseExpGrowth` - the base increment exp number of this skill
```

Comment:
```suggestion
- `baseExpGrowth` - the base experience increment number of this skill
```

Reply:
Don't short words in description sentences, as this can reduce clarity and lead to confusion.